fbot_world_bringup/nodes/pose.py

```diff
@@ -42,18 +42,19 @@ def __init__(self, node_name: str = 'pose'):
     self.get_logger().info(f"File name: {self.config_file_name}")
 
     self.setStaticPose()
-    self.pose_server = self.create_service(GetPose, '/fbot_world/get_pose', self.getPose)
+    self.pose_server = self.create_service(GetPose, '/fbot_world/get_pose', self.getTargets)
```

Comment:
```suggestion
    self.pose_server = self.create_service(GetPose, '/fbot_world/get_pose', self.getPose)
```
No need for an intermediate function anymore.